docker-compose.yaml

```diff
@@ -160,5 +164,50 @@ services:
     ports:
       - 9090:9090
 
+  es:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
+    container_name: es
+    hostname: "es.${CONTAINER_DOMAIN}"
+    networks:
+      - vault-playground
+    volumes:
+      - ./docker/es:/usr/share/elasticsearch/data:Z
+    ports:
+      - 9200:9200
+    environment:
+      - discovery.type=single-node
+      - xpack.security.enabled=false
```

Comment:
we disable the Elasticsearch password authentication on the API here, since this is a playground (minimal config)